docs/development.md

```diff
@@ -12,72 +12,72 @@ Here are a few things to know to get you started on the right path.
 
 ## Branching
 
-Mermaid uses a [Git Flow](https://guides.github.com/introduction/flow/) inspired approach to branching. So development is done in the `develop` branch.
+Mermaid uses a [Git Flow](https://guides.github.com/introduction/flow/)–inspired approach to branching. So development is done in the `develop` branch.
 
-Once development is done we branch a release branch from develop for testing.
+Once development is done we branch a `release` branch from `develop` for testing.
 
-Once the release happens we merge the release branch with master and kill the release branch.
-
-This means... **you should branch off your pull request from develop** and direct all Pull Requests to it.
+Once the release happens we merge the `release` branch with `master` and kill the `release` branch.
 
+This means that **you should branch off your pull request from develop** and direct all Pull Requests to it.
 
 ## Contributing Code
 
-We make all changes via pull requests. As we have many pull requests from developers new to mermaid, we have put in place a process, wherein *knsv, Knut Sveidqvist* is the primary reviewer of changes and merging pull requests. It is as follows:
+We make all changes via Pull Requests. As we have many Pull Requests from developers new to mermaid, we have put in place a process, wherein *knsv, Knut Sveidqvist* is the primary reviewer of changes and merging pull requests. The process is as follows:
 
 * Large changes reviewed by knsv or other developer asked to review by knsv
-* Smaller low-risk changes like dependecies, documentation etc can be merged by active collaborators
-* documentation (updates to the docs folder are enocouraged and also allowed via direct commits)
+* Smaller, low-risk changes like dependecies, documentation, etc. can be merged by active collaborators
+* Documentation (updates to the docs folder are enocouraged and also allowed via direct commits)
```

Comment:
enocourage or encouraged?

Reply:
thanks, I committed a new change to use active voice (encourage instead of encouraged)